WalletWasabi.Fluent/CrashReport/ViewModels/CrashReportWindowViewModel.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.IO;
+using ReactiveUI;
+using System.Windows.Input;
+using WalletWasabi.Fluent.ViewModels.HelpAndSupport;
+using WalletWasabi.Models;
+using WalletWasabi.Gui.ViewModels;
+
+namespace WalletWasabi.Fluent.CrashReport.ViewModels
+{
+	public partial class CrashReportWindowViewModel : ViewModelBase
+	{
+		[AutoNotify] private SerializableException _serializableException;
+		[AutoNotify] private string _logPath;
+
+		public CrashReportWindowViewModel(SerializableException exception, string logPath)
+		{
+			SerializableException = exception;
+			LogPath = logPath;
+			CancelCommand = ReactiveCommand.Create(CrashReporter.RestartWasabi);
+			NextCommand = ReactiveCommand.Create(CrashReporter.ShutdownWasabi);
+			OpenGitHubRepoCommand = ReactiveCommand.CreateFromTask(async () =>
+			{
+				await IoHelpers.OpenBrowserAsync(AboutViewModel.BugReportLink);
```

Comment:
The link is wrong. It should open the support page, IMO this https://github.com/zkSNACKs/WalletWasabi/discussions/5185. Please elaborate on that, I am not sure where is the problem.

Reply:
done! :)